### ISSUE TITLE
Error: Error creating DB Subnet Group: DBSubnetGroupAlreadyExists: The DB subnet group 'my-cool-project-vpc' already exists.

### ISSUE THREAD

**Issue by maina360** [terraform-in-action/manning-code#20]
Getting the following error when I ran the complete version on ch4. This is the first time running the code!

 Error: Error creating DB Subnet Group: DBSubnetGroupAlreadyExists: The DB subnet group 'my-cool-project-vpc' already exists.
│ 	status code: 400, request id: 7e127cce-77c6-42fb-b52d-a0db080e72dc
│ 
│   with module.networking.module.vpc.aws_db_subnet_group.database[0],
│   on .terraform/modules/networking.vpc/main.tf line 408, in resource "aws_db_subnet_group" "database":
│  408: resource "aws_db_subnet_group" "database" {
│

**Comment by scottwinkler** [comment]
I suspect this is a user error. There is no reason for the DB subnet group with that name to exist unless it was deployed either by yourself or another person in your AWS account. Are you sure the code is exactly the same as what I have here?

**Comment by maina360** [comment]
Yes it is, I will try this in a different region and also double check if
someone else ran the code. Thank you!

**Comment by maina360** [comment]
So I found out what the issue was. When I first ran the code, I got this error that need me to add  sensitive = true to the output in the root module outputs.tf

output "db_password" {
  sensitive = true
  value = module.database.db_config.password
}

After seeing that error I ran, destroy command but the RDS subnet group never got destroyed, not sure why and that was causing the issue I opened. Anyway, after figuring that out there was another error with the resource "random_password" "password"  for this one I had to remove the extra \ on there to fix the issue then I ran a destroy again and apply and this time everything worked well. So you might want to modify the two resources I mentioned for future users. Thank you!
